src/mission.cpp

```diff
@@ -2496,6 +2496,32 @@ Before we get into a major fight just make sure we have the gear we need, boss."
         }
         break;
 
+    case MISSION_FIND_ZSG:
+        if( state == "TALK_MISSION_DESCRIBE" ) {
+            return _("I've can an idea about how to improve our chances of surviving");
```

Comment:
"can an idea"? Maybe "I've an idea..." - the "can" seems wrong there. There seems to be a period missing at the end of the sentence.